pallets/proposals/src/impls.rs

```diff
@@ -439,7 +440,33 @@ impl<T: Config> Pallet<T> {
             now,
         ));
 
+        //once the voting is complete check if the milestone is eligible for auto approval
+        //Getting the total threshold required for the milestone to be approved based on the raised funds
+        let funding_threshold: BalanceOf<T> = project.raised_funds * T::PercentRequiredForVoteToPass::get().into()/100u32.into();
+
+        let mut milestone = project.milestones.get_mut(&milestone_key).unwrap().clone();
```

Comment:
No unwraps in runtime code please.

Reply:
Also no need to clone as you are taking a mutable reference to edit the milestone.

See the below comments